doc/changelog/07-commands-and-options/13352-cep-48.rst

```diff
@@ -0,0 +1,12 @@
+- **Changed:**
+  Option -native-compiler of the configure script now impacts the
+  default value of the option -native-compiler of coqc. The
+  -native-compiler option of the configure script is added an ondemand
+  value, which becomes the default, thus preserving the previous default
+  behavior.
+  The stdlib is still precompiled when configuring with -native-compiler
+  yes. It is not precompiled otherwise.
+  This an implementation of point 2 of
+  `CEP #48 <https://github.com/coq/ceps/pull/48>`
```

Comment:
Sorry again @proux01! but the final `_` also seems necessary:
https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#hyperlinks

Reply:
Indeed, thanks!